pkg/informer/informer.go

```diff
@@ -30,6 +43,50 @@ type GenericInformer struct {
 	retries       int64             // Counts times we have tried without establishing a watch.
 }
 
+///////////////////////////////////////////////////////////////////
+
+//leaving this here incase we want it to use other fields in future:
+
+// type Config struct {
+// 	APIVersion string `yaml:"apiVersion"`
+// 	Kind       string `yaml:"kind"`
+// 	Metadata   struct {
+// 		Name      string `yaml:"name"`
+// 		Namespace string `yaml:"namespace"`
+// 	} `yaml:"metadata"`
+// 	Data struct {
+// 		AllowedResources string `yaml:"AllowedResources"`
+// 		DeniedResources  string `yaml:"DeniedResources"`
+// 	} `yaml:"data"`
+// }
+
+type AllowedResources struct {
+	ApiGroups []string `yaml:"apiGroups"`
+	Resources []string `yaml:"resources"`
+}
+
+type DeniedResources struct {
+	ApiGroups []string `yaml:"apiGroups"`
+	Resources []string `yaml:"resources"`
+}
+
+func GetAllowDenyData(cm *v1.ConfigMap) ([]AllowedResources, []DeniedResources) {
+
+	var allow []AllowedResources
+	allowerr := yaml.Unmarshal([]byte(cm.Data["AllowedResources"]), &allow)
+	if allowerr != nil {
+		klog.Fatalf("Unmarshal: %v", allowerr)
```

Comment:
`klog.Fatal()` will exit the go program.  Let's lower these to `klog.Error()`.  We should tolerate this error and continue with default values.